### ISSUE TITLE
Upload to steam

### ISSUE THREAD

**Issue by iEzioC** [henrypp/memreduct#221]
He! I really love your software, just one question. Would you mind upload your software to Steam app? Which is a platform with tons of games and softwares (such as obs) on it. It would be perfect to see your software on that platform!

**Comment by Sahil12524** [comment]
No, I think this app must be re written in better way and must rework on modern UI the Win32 UI looks kinda old fashion. However, the developer can make modern UI using WinUI 3.

**Comment by micxdev** [comment]
> No, I think this app must be re written in better way and must rework on modern UI the Win32 UI looks kinda old fashion. However, the developer can make modern UI using WinUI 3.

Come on mate, WinUI 3 sucks! Native win32 ui for this program is perfectly acceptable and much better. Having that extra performance overhead is not worth it.

**Comment by Sahil12524** [comment]
> > No, I think this app must be re written in better way and must rework on modern UI the Win32 UI looks kinda old fashion. However, the developer can make modern UI using WinUI 3.
> 
> Come on mate, WinUI 3 sucks! Native win32 ui for this program is perfectly acceptable and much better. Having that extra performance overhead is not worth it.

Yeah, I understand. My bad!
